source/targets/mas_steekproef_zl_lm_za_kp_po/_targets_r/targets/raw-data.R

```diff
@@ -42,10 +42,10 @@ list(
   ),
   tarchetypes::tar_file(
     existing_file,
-    path_to_existing(file = "steekproef_piloot_avimap.geojson")
+    path_to_existing(file = "avimap_601_0_MAS_Vlaanderen_telpunten_xy.shp")
```

Comment:
Ik vermoed dat dit inderdaad een bestand is dat rechtstreeks van Sovon komt en dus mogelijk met die specifieke projectie pipeline moet omgezet worden